apis/core/v1beta1/loggingcluster_types.go

```diff
@@ -39,10 +39,11 @@ type LoggingCluster struct {
 
 type LoggingClusterSpec struct {
 	OpensearchClusterRef *opnimeta.OpensearchClusterRef `json:"opensearchCluster,omitempty"`
-	IndexUserSecret      *corev1.LocalObjectReference   `json:"indexUser,omitempty"`
-	FriendlyName         string                         `json:"friendlyName,omitempty"`
-	LastSync             metav1.Time                    `json:"lastSync,omitempty"`
-	Enabled              bool                           `json:"enabled,omitempty"`
+	// Deprecated field
```

Comment:
I think if you format this comment like `// Deprecated: reason` it will show up correctly in IDEs